UnityProject/Assets/Scripts/Systems/Antagonists/AlienPlayer.cs

```diff
@@ -544,8 +544,12 @@ private void PlasmaCheck()
 			//Don't need to check if full
 			if(currentPlasma == alienType.MaxPlasma) return;
 
-			//Need to be on weeds
-			if(onWeeds == false) return;
+			//Need to be on weeds for full plasma gain
+			if (onWeeds == false)
+			{
+				TryAddPlasma(1);
```

Comment:
Why magical number and not add the minimum to the constants you are using in the rest of the code?